PROVESFlightControllerReference/test/format_filesystem.py

```
@@ -0,0 +1,38 @@
+    while time.time() < timeout_time:
+        try:
+            fprime_test_api_session.send_and_assert_command(
+                command=f"{cmdDispatch}.CMD_NO_OP"
+            )
+            gds_working = True
+            break
+        except Exception:
+            time.sleep(1)
+    assert gds_working
+
+    yield
```

Ruff (0.15.12)

[warning] No teardown in fixture `check_gds`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)